### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#8738] [Help]
Hello everyone.
I created an automation **(cron trigger)** that uses **(query rows)** to get rows from a table with a **filter (status=waiting).**
the question is how to **modify all these rows(statut=waiting)**? I think we should use **(add looping)** but I don't know how to do it.
@melohagan

**Comment by melohagan** [comment]
Hey @rafikApp 

Does this help? https://docs.budibase.com/docs/looping

What have you tried - is this related to your other post?
